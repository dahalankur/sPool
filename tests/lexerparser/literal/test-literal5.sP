# lists
[1, 2, 3]
[1, true, 3.45, abc, "heterogenous lists are ok right now...but semantically? never."]
[]
