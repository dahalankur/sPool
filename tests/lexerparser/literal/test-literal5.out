PROGRAM {[ 
    LIST(INT(1), INT(2), INT(3)) 
    LIST(INT(1), BOOL(true), FLOAT(3.45), VAR(abc), STRING("heterogenous lists are ok right now...but semantically? never.")) 
    LIST()]}
